Core/src/org/sleuthkit/autopsy/modules/drones/DroneIngestModuleFactory.java

```diff
@@ -33,8 +33,8 @@
 public class DroneIngestModuleFactory extends IngestModuleFactoryAdapter {
 
     @Messages({
-        "DroneIngestModule_Name=Drone",
-        "DroneIngestModule_Description=Description"
+        "DroneIngestModule_Name=Drone Analyzer",
+        "DroneIngestModule_Description=Analyzes file generated by drones."
```

Comment:
I think "files" should be plural in the module description.

Reply:
Correct.